test/xbgpu/test_engine.py

```diff
@@ -770,16 +792,19 @@ async def _send_data(
             Timestamp step between each received heap processed.
         n_channels_per_substream, n_spectra_per_heap, frequency
             See :meth:`_create_heaps` for more info.
+        beam_capture_stop_heap_index
+            Heap count at which a `?capture-stop` was issued to beam streams.
 
         Returns
         -------
         corrprod_results
-            List of arrays of all GPU-generated data. One output array per
-            corrprod_output, where each array has shape
+            Dictionary of arrays of all XPipeline output. Each key is the
+            corrprod_output name. Each value is an array with shape
             (n_accumulations, n_channels_per_substream, n_baselines, COMPLEX).
         beam_results
-            Beamformer output, with shape (n_beams, n_batches,
-            n_channels_per_substream, n_spectra_per_heap, COMPLEX).
+            Dictionary of arrays of all BPipeline output. Each key is the
+            beam_output name. Each value is an array with shape
+            (n_batches, n_channels_per_substream, n_spectra_per_heap, COMPLEX).
```

Comment:
I suppose it's not entirely necessary for **both** of these returned results to be dictionaries anymore - now that we are only `capture-stop`'ing the beam data streams.

I've left it as-is just to keep it homogeneous. It will likely cause confusion, and generally just look odd, if the results are different data types/orientations.